extensions/ql-vscode/src/debugger/debug-session.ts

```diff
@@ -333,6 +392,50 @@ export class QLDebugSession extends LoggingDebugSession implements Disposable {
     super.stackTraceRequest(response, _args, _request);
   }
 
+  protected customRequest(
+    command: string,
+    response: CodeQLDebugProtocol.Response,
+    args: any,
+    request?: DebugProtocol.Request,
+  ): void {
+    switch (command) {
+      case "codeql-quickeval": {
+        this.quickEvalRequest(
+          response,
+          <CodeQLDebugProtocol.QuickEvalRequest["arguments"]>args,
```

Comment:
We're generally using this style for type casts. 

```suggestion
          args as CodeQLDebugProtocol.QuickEvalRequest["arguments"],
```

Reply:
Fixed